library/Reporting/Scheduler.php

```diff
@@ -149,27 +149,23 @@ protected function add(Schedule $schedule)
         }
 
         $next = clone $start;
```

Comment:
This is fully obsolete once you're using the current time inside the timer again. Remove this and don't pass it on.

Reply:
No it's not, I've had not enough coffee yet -.-